src/components/Result.vue

```diff
@@ -74,6 +34,11 @@
           return lowerPokemon.includes(lowerSearch);
         },
       },
+      mounted() {
+        PokemonService.getPokemons().then(pokemons => {
+          this.pokemons = pokemons;
+        });
+      }
```

Comment:
You can use async / await in the mounted hook. I think it could be nice to show / explain during the demo